components/automate-ui/src/app/page-components/user-table/user-table.component.html

```diff
@@ -1,47 +1,33 @@
 <app-authorized [allOf]="getPermissionsPath" [overrideVisible]="overridePermissionsCheck">
-  <ng-container *ngIf="(users$ | async)?.length === 0">
```

Comment:
we wanted to use the loading status in addition to the number of users to determine if this empty state message should be shown, so we decided to move the empty message into the parent components, which has access to that loading variable